pyspecProcScripts/correlation_alignment.py

```diff
@@ -170,7 +170,8 @@ def correl_align(
                     ax=ax_list[1],
                 )
                 ax_list[1].set_title("correlation function (t), \nafter apod")
-        correl.ft_clear_startpoints(direct)
+        correl.ft_new_startpoint(direct, "freq")
+        correl.ft_new_startpoint(direct, "time")
```

Comment:
reset startpoints for time AND frequency domain

Reply:
This just uses the new function. The point of the new function is that the source is simpler and more readable because it does one domain at a time